### ISSUE TITLE
Accessibility isssue on Menu  

### ISSUE THREAD

**Issue by allinagaraju** [primefaces/primeng#5198]
Screen reader does not read the menu items properly.
 
MAS Standard Failed : 4.1.2 - Name, Role, Value
1)When the focus is on the Signed-in username button menu, press enter.
2) Use tab keys to navigate through the menu items,
3) Observe what the screen raeder reads when the menu items receive focus.
Error: Screen reader reads junk data such as "N", "100 link" and "s" along wit the actual data as shown in the screenshot.
![large](https://user-images.githubusercontent.com/30447312/36517830-aa3e2b44-1738-11e8-8e62-a69c7a8358ac.png)

Could you please help us in understanding , how to overcome the issue or removing Junk Characters such as "N", "100 link" and "s" along wit the actual data

**Comment by allinagaraju** [comment]
Any Updates Please

**Comment by jason-jason-h** [comment]
This will be a good enhancement to see!

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
